express_cut/express_service/urls.py

```diff
@@ -41,6 +41,7 @@
 
     path('stylist', view_role.get_all_stylists),
     path('stylist/<int:stylist_id>/reservation', view_reservation.reservations_by_stylist),
+    path('stylist/<int:stylist_id>/reservedslots', view_reservation.get_daily_reservations),
```

Comment:
Lets use camelCase here for the reservedSlots route or simply just go back to reserved should be good as well

Reply:
Will change to reservedTimeSlots.